<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 219, in render
    yaml_text = _get_template(meta_yaml, selector_dict).render(render_dict)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 155, in _get_template
    return _jinja_silent_undef.from_string(meta_yaml_selectors_applied)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/jinja2/environment.py", line 1105, in from_string
    return cls.from_code(self, self.compile(source), gs, None)
                               ^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/jinja2/environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<unknown>", line 1, in template
jinja2.exceptions.TemplateSyntaxError: unexpected '}'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 722, in lint_file
    recipe = _recipe.Recipe.from_file(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 231, in from_file
    raise exc
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 222, in from_file
    recipe._load_from_string(text.read())
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 154, in _load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 271, in render
    self.meta = renderer.render(
                ^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/_renderer.py", line 221, in render
    raise JinjaRenderFailure(recipe_dir, message=exc.message, line=exc.lineno)
percy.render.exceptions.JinjaRenderFailure: (PosixPath('/tmp/abs_8ewa40f_j1/clone/recipe'), "unexpected '}' (at line 1)")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
             ^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/run.py", line 114, in main
    result = linter.lint(
             ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 626, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/anaconda_linter/lint/__init__.py", line 739, in lint_file
    recipe = _recipe.Recipe(recipe_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.11/site-packages/percy/render/recipe.py", line 93, in __init__
    self.recipe_dir = recipe_file.parent
                      ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'parent'

ERROR conda.cli.main_run:execute(47): `conda run conda-lint /tmp/abs_8ewa40f_j1/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -31,7 +31,27 @@ requirements:
     - click >=7.0
     - black >=19.10b0
     - rich >=9.0.0
-
+  run_constrained:
+    # codemods
+    - libcst >=0.3.16
+    # pytz
+    - pytz >=2014.1
+    #dateutil
+    - python-dateutil >=1.4
+    #lark
+    - lark >=0.10.1
+    #numpy
+    - numpy >=1.17.3
+    #pandas
+    - pandas >=1.1
+    #pytest
+    - pytest >=4.6
+    #dpcontracts
+    #- dpcontracts >=0.4 #Not available currently
+    #redis
+    - redis-py >=3.0.0
+    #django
+    - django >=3.2
 test:
   requires:
     - django

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,4 +1,4 @@
-{% set name = "hypothesis"}
+{% set name = "hypothesis" %}
 {% set version = "6.82.0" %}
 
 package:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -54,7 +54,6 @@ requirements:
     - django >=3.2
 test:
   requires:
-    - django
     - pandas
     - pip
     - python
@@ -62,7 +61,6 @@ test:
   imports:
     - hypothesis
     - hypothesis.extra
-    - hypothesis.extra.django
     - hypothesis.extra.pandas
     - hypothesis.internal
     - hypothesis.internal.conjecture

